Automattic/wp-calypso
# Backup: Implement backup staging flag update API

Similar to #75856, lets add the necessary logic to implement the new API for marking sites as staging (D107983-code).

## Proposed Changes
* Add necessary handlers and redux logic to implement new `POST rewind/staging/update` API to fetch and mark sites as staging.

## To-do
- [x] Action types (API request life cycle)
- [x] API fetch handler
- [x] Reducers
- [x] Selectors 
- ~Data query component (if needed)~

## Next steps
* Implement handlers and redux logic for `GET rewind/staging` endpoint (D107983-code).
* Integrate with the new mark a site as Staging Site Jetpack setting.

## Testing Instructions
This is an isolated change that will be integrated on the Jetpack settings page. So at this point you can ensure the unit tests are passing by using `yarn run test-client client/state/rewind`.

## Pre-merge Checklist
